mod/bookmarks/start.php

```diff
@@ -44,79 +42,6 @@ function bookmarks_init() {
 	elgg_register_plugin_hook_handler('likes:is_likable', 'object:bookmarks', 'Elgg\Values::getTrue');
 }
 
-/**
- * Dispatcher for bookmarks.
- *
- * URLs take the form of
- *  All bookmarks:        bookmarks/all
- *  User's bookmarks:     bookmarks/owner/<username>
- *  Friends' bookmarks:   bookmarks/friends/<username>
- *  View bookmark:        bookmarks/view/<guid>/<title>
- *  New bookmark:         bookmarks/add/<guid> (container: user, group, parent)
- *  Edit bookmark:        bookmarks/edit/<guid>
- *  Group bookmarks:      bookmarks/group/<guid>/all
- *  Bookmarklet:          bookmarks/bookmarklet/<guid> (user)
- *
- * Title is ignored
- *
- * @param array $page URL segments
- * @return bool
- */
-function bookmarks_page_handler($page) {
-
-	if (!isset($page[0])) {
-		$page[0] = 'all';
-	}
-
-	elgg_push_breadcrumb(elgg_echo('collection:object:bookmarks'), 'bookmarks/all');
```

Comment:
move this breadcrumb to the resource views

Reply:
`elgg_push_collection_breadcrumbs()`